### PR TITLE
Add message id to billing transaction table

### PR DESCRIPTION
**Pull request by hodgestar**

### CODE REVIEW

**Comment by hodgestar** [comment]
Ready for review.

File: go/billing/api.py [hunk 1]
```diff
@@ -578,14 +578,16 @@ def render_POST(self, request):
...
             account_number = data.get('account_number', None)
+            message_id = data.get('message_id', None)
             tag_pool_name = data.get('tag_pool_name', None)
             tag_name = data.get('tag_name', None)
             message_direction = data.get('message_direction', None)
             session_created = data.get('session_created', None)
-            if (account_number and tag_pool_name and tag_name
+            if (account_number and message_id
+                    and tag_pool_name and tag_name
                     and message_direction and session_created is not None):
```
Comment (justinvdm) [inline go/billing/api.py]: These `and`s are stacking up pretty quick. Probably best left this way for now, just wondering of better ways for later.

File: go/billing/models.py [hunk 2]
```diff
@@ -148,6 +148,11 @@ class Transaction(models.Model):
     tag_name = models.CharField(max_length=100, blank=True)
     message_direction = models.CharField(max_length=20, blank=True)
 
+    message_id = models.CharField(
+        max_length=64, null=True, blank=True,
+        help_text=_("Vumi message identifier for the message being"
+                    " billed (or null if there is no assocated message)"))
```
Comment (justinvdm) [inline go/billing/models.py]: typo - 'assocated'

**Comment by justinvdm** [comment]
Other than the typo, +1

**Comment by hodgestar** [comment]
Ready for re-review.

**Comment by justinvdm** [comment]
:+1: not sure if we should wait for travis, it seemed happy before the last two commits, and the last two commits were pretty minor. Your call.

**Pull request merged**